unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -84,17 +121,7 @@ public async UniTask<FriendRequest> RequestFriendshipAsync(string friendUserId,
         {
             cancellationToken.ThrowIfCancellationRequested();
 
-            RequestFriendshipConfirmationPayload payload = await apiBridge.RequestFriendshipAsync(friendUserId, messageBody, cancellationToken);
```

Comment:
no changes needed here if you keep the interface `IFriendsApiBridge` contract, specially for the feature flag and retro-compatibility issues